pallets/eth-bridge/src/benchmarking.rs

```diff
@@ -0,0 +1,167 @@
+//! # Eth bridge pallet
+// Copyright 2023 Aventus Network Services (UK) Ltd.
+
+//! eth-bridge pallet benchmarking.
+
+#![cfg(feature = "runtime-benchmarks")]
+
+use crate::*;
+
+use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite};
+use frame_support::{ensure, BoundedVec};
+use frame_system::RawOrigin;
+use sp_core::H256;
+use sp_runtime::WeakBoundedVec;
+
+fn setup_authors<T: Config>(number_of_validator_account_ids: u32) -> Vec<crate::Author<T>> {
+    let mnemonic: &str =
+        "basic anxiety marine match castle rival moral whisper insane away avoid bike";
+    let mut authors: Vec<crate::Author<T>> = Vec::new();
+    for i in 0..number_of_validator_account_ids {
+        let account = account("dummy_validator", i, i);
+        let key =
+            <T as avn::Config>::AuthorityId::generate_pair(Some(mnemonic.as_bytes().to_vec()));
+        authors.push(crate::Author::<T>::new(account, key));
+    }
+
+    // Setup authors in avn pallet
+    avn::Validators::<T>::put(WeakBoundedVec::force_from(
+        authors.clone(),
+        Some("Too many authors for session"),
```

Comment:
Why using a `WeakBoundedVec` and is it possible to have too many authors for session?